desc/compute/data_index.py

```diff
@@ -45,18 +46,21 @@ def register_compute_fun(
         Coordinate dependency. IE, "rtz" for a function of rho, theta, zeta, or "r" for
         a flux function, etc.
     data : list of str
-        Names of other items in the data index needed to compute qty
+        Names of other items in the data index needed to compute qty.
+    limit_data : list of str
```

Comment:
`limit_data` could be ambiguous, maybe `axis_data` or `axis_limit_data`?

Reply:
i'll pick the latter